### DISCUSSION TITLE
Multi variable length argument

### DISCUSSION THREAD

**Discussion by ghost** [vlang/v#18604] [Ideas]
```v
fn foo(data ...(int, string)){
}

fn main(){
	foo(1, 'A', 2, 'B', 3, 'C')
}
```

**Comment by JalonSolov** [comment]
Your first example looks like a tuple, which V doesn't support, and there are no plans to do so.

**Comment by IkeVoodoo** [comment]
I believe this is more of a compound type.

Data is a vararg that accepts both numbers and strings.

**Comment by JalonSolov** [comment]
V has sum types for that.
